### ISSUE TITLE
Making SIlver Hand playable

### ISSUE THREAD

**Issue by masteofchaos** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#470]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
**Describe your suggestion in full detail below:**
Helllo i'd like to suggest a  rework for the Silver Hand. As of right now the bulwark against the undead and it's sucessors (scarlet crusade ans argent dawn) are unplayable (unless you start as a vassal of them). This is very sad as i'd love to be playing as uther and defending the last remnants of Lordaeron against the UNdead. My suggestion is simple: make the Silver Hand into a titular lordship and then make a holy order called Paladins of the silver hand if you are unwilling to let go of the Silver Hand's representation as a holy order. Furthermore the same treatment should be given to the Argent Dawn and the Scarlet Crusade.  Then you can give the Scarlet crusade the Crimson Legion as a holy order and you can give the Argent Dawn the Brotherhood of the light. One final thing (ik this is already quite a rework but if you're willing) make it so that the Scarlet crusade and the Argent dawn be lead by their historical leaders instead of two random dudes that just happen to spawn. Or make it two seperate decisions so the player himself can form them themselves. 
Thank you very much if you read this and i hope to see these features implemented
Keep up the masterful work!

**Comment by zumbak04** [comment]
They're already playable right now :3
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/60680523-bb8a4a80-9e9c-11e9-8dc5-b838d90d12cd.png)

</details>

**Comment by masteofchaos** [comment]
@zumbak04  well thats wierd last time i started the game it didn't allow me to play because it's a holy order.

**Comment by zumbak04** [comment]
> @zumbak04 well thats wierd last time i started the game it didn't allow me to play because it's a holy order.

I used `play` command and `charinfo` to found out character ID.
https://ck2.paradoxwikis.com/Console_commands

**Comment by masteofchaos** [comment]
Oh so that's the only way. Ok thanks. How about for Scarlet and Argent Dawn does it work the same way?

**Comment by zumbak04** [comment]
> Oh so that's the only way. Ok thanks. How about for Scarlet and Argent Dawn does it work the same way?

Yep, it works the same even for vanilla holy orders. The game blocks picking character with holy order titles, but doesn't kick players.

**Comment by masteofchaos** [comment]
@zumbak04 .thank you but i still think you should change it so that it can be played in multiplayer games. You can't rly use cheats in ck2 mp

**Comment by zumbak04** [comment]
> @zumbak04 .thank you but i still think you should change it so that it can be played in multiplayer games. You can't rly use cheats in ck2 mp

Mmmkay, but it requires some time. The best way is to give holy order title after the game starts, BUT! If you're going to load a save after, you won't be able to pick it.

**Comment by masteofchaos** [comment]
As long as it is eventually done it would be great. Thanks anyway.
